spec/models/mixins/relationship_mixin_spec.rb

```diff
@@ -216,6 +216,7 @@ def self.name
           has_ancestry
           self.table_name = "services"
           include RelationshipMixin
+          self.default_relationship_type = "custom"
```

Comment:
this doesn't look necessary. like having it in there but could you share why it was necessary (so I can learn)?

Reply:
These tests fail without it.

Reply:
can you change the test to use the name/table from a class that has `RelationshipMixin` included. Something like `host`.

For that model, is the `default _relationship_type` still needed?

Reply:
Hosts ... don't have ancestry? So, no? Not yet, anyway.